mlrun/model_monitoring/db/stores/__init__.py

```diff
@@ -16,60 +16,61 @@
 
 import enum
 import typing
+import warnings
 
 import mlrun.common.schemas.secret
 import mlrun.errors
 
-from .model_endpoint_store import ModelEndpointStore
+from .base import StoreBase
 
 
-class ModelEndpointStoreType(enum.Enum):
-    """Enum class to handle the different store type values for saving a model endpoint record."""
+class ObjectStoreType(enum.Enum):
+    """Enum class to handle the different store type values for saving model monitoring records."""
 
     v3io_nosql = "v3io-nosql"
     SQL = "sql"
 
-    def to_endpoint_store(
+    def to_object_store(
         self,
         project: str,
         access_key: str = None,
-        endpoint_store_connection: str = None,
+        store_connection: str = None,
         secret_provider: typing.Callable = None,
-    ) -> ModelEndpointStore:
+    ) -> StoreBase:
         """
-        Return a ModelEndpointStore object based on the provided enum value.
+        Return a StoreBase object based on the provided enum value.
 
-        :param project:                    The name of the project.
-        :param access_key:                 Access key with permission to the DB table. Note that if access key is None
-                                           and the endpoint target is from type KV then the access key will be
-                                           retrieved from the environment variable.
-        :param endpoint_store_connection: A valid connection string for model endpoint target. Contains several
+        :param project:                   The name of the project.
+        :param access_key:                Access key with permission to the DB table. Note that if access key is None
+                                          and the endpoint target is from type KV then the access key will be
+                                          retrieved from the environment variable.
+        :param store_connection:          A valid connection string for the store target. Contains several
                                           key-value pairs that required for the database connection.
                                           e.g. A root user with password 1234, tries to connect a schema called
                                           mlrun within a local MySQL DB instance:
-                                          'mysql+pymysql://root:1234@localhost:3306/mlrun'.
+                                          'mysql+pymysql://root:1234@localhost:3306/mlrun_model_monitoring'.
         :param secret_provider:           An optional secret provider to get the connection string secret.
 
-        :return: `ModelEndpointStore` object.
+        :return: `StoreBase` object.
 
         """
 
-        if self.value == ModelEndpointStoreType.v3io_nosql.value:
-            from .kv_model_endpoint_store import KVModelEndpointStore
+        if self.value == ObjectStoreType.v3io_nosql.value:
```

Comment:
Suggestion - why not simply:
```suggestion
        if self == self.v3io_nosql:
```